src/BuildingBlocks/Globalization/Masa.BuildingBlocks.Globalization.I18n.Tests/CustomResource.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Masa.BuildingBlocks.Ddd.Domain.Tests;
 
-public class CustomerResource
+public class CustomResource
```

Comment:
file name

Reply:
The file name has been changed, you may need to refresh it